packages/core/src/Lib/UseFlyonUIVueAppConfig/Internal/Lib/UseFlyonUIVueAppConfigProperty.ts

```
@@ -0,0 +1,26 @@
+import type {
+    ConfigurableComponentName,
+    ConfigurableProperty,
+    FlyonUIVueAppComponentsConfig,
+    FlyonUIVueAppDefaultConfig,
+    FlyonUIVueAppGlobalConfig,
+} from '@/Lib';
+
+import type { ComputedRef, MaybeRefOrGetter } from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/Lib" (internal-type).

(perfectionist/sort-imports)